.ci-scripts/release.bash

```diff
@@ -109,24 +103,25 @@ result=$(curl -X POST "https://api.github.com/repos/${GITHUB_REPOSITORY}/release
 rslt_scan=$(echo "${result}" | jq -r '.id')
 if [ "$rslt_scan" != null ]
 then
-  echo "Release notes uploaded"
+  echo -e "\e[34mRelease notes uploaded\e[0m"
 else
-  echo "Unable to upload release notes, here's the curl output..."
+  echo "\e[31mUnable to upload release notes, here's the curl output...\e[0m"
   echo "${result}"
   exit 1
 fi
 
+# pull again, just in case, odds of this being needed are really slim
+git pull
+
 # update CHANGELOG for new entries
-git checkout master
-git merge "prep-release-${VERSION}"
+echo -e "\e[34mAdding new 'unreleased' section to CHANGELOG.md\e[0m"
 changelog-tool unreleased -e
 
 # commit changelog and push to master
+echo -e "\e[34mCommiting CHANGELOG.md change\e[0m"
 git add CHANGELOG.md
-git commit -m "Add unreleased section to CHANGELOG post ${VERSION} release prep
-
-[skip ci]"
-git push ${PUSH_TO} master
+git commit -m "Add unreleased section to CHANGELOG post ${VERSION} release [skip ci]"
 
 # delete release-VERSION tag
+echo -e "\e[34mDeleting no longer needed remote tag release-${VERSION}\e[0m"
 git push --delete ${PUSH_TO} "release-${VERSION}"
```

Comment:
Just to understand the process, this tag is created manually in a previous step?

Reply:
Yes, see the release process document.

Reply:
:heavy_check_mark: